packages/vite/client.d.ts

```diff
@@ -198,6 +198,11 @@ declare module '*?worker&inline' {
   export default workerConstructor
 }
 
+declare module '*?worker&url' {
+  const src: string
+  export default src
+}
+
 declare module '*?sharedworker' {
```

Comment:
Maybe we can also add the types for `*?sharedworker&inline` and `*?sharedworker&url` too